### PR TITLE
Loosen dependency versions

### LINKED ISSUE #942: Can't use latest Alamofire and RxSwift with Moya 8.0.1
I have the following podfile:
  pod 'Alamofire', '\~> 4.3'
  pod 'RxSwift', '\~> 3.2'
  pod 'RxCocoa', '\~> 3.2'
  pod 'Moya/RxSwift', '8.0.1'

When I run pod update I get:
[!] Unable to satisfy the following requirements:
- `Alamofire (~> 4.3)` required by `Podfile`
- `Alamofire (~> 4.2.0)` required by `Moya/Core (8.0.1)`
- `RxSwift (~> 3.2)` required by `Podfile`
- `RxSwift (~> 3.2)` required by `Podfile`
- `RxSwift (~> 3.1)` required by `RxCocoa (3.2.0)`
- `RxSwift (~> 3.1.0)` required by `Moya/RxSwift (8.0.1)`

### PR DESCRIPTION
**Pull request by BasThomas**
Fixes #942

### CODE REVIEW

**Comment by bjarkehs** [comment]
I guess we don't rely on functionality from `4.1`, or `4.2` of `Alamofire`. Or `3.1` for `RxSwift`. Else I'd suggest making the dependencies: `"Alamofire", "~> 4.2"` and `"RxSwift", "~> 3.1"`

**Comment by BasThomas** [comment]
Good point @bjarkehs; we actually rely on Alamofire 4.1 (https://github.com/Moya/Moya/pull/797)

File: Moya.podspec [hunk 1]
```diff
@@ -34,12 +34,12 @@ Pod::Spec.new do |s|
   s.subspec "ReactiveSwift" do |ss|
     ss.source_files = "Sources/ReactiveMoya/"
     ss.dependency "Moya/Core"
-    ss.dependency "ReactiveSwift", "1.0.0"
+    ss.dependency "ReactiveSwift", "1.0"
```
Comment (pedrovereza) [inline Moya.podspec]: Should this be `~> 1.0` as well? 🤔

Comment (BasThomas) [inline Moya.podspec]: Good catch!

**Comment by pedrovereza** [comment]
Changes seem good 💯 

Should we plan a new release or is it too soon?

**Comment by ashfurrow** [comment]
Makes sense to do a patch release I think 👍

**Pull request merged**